projects/common/03oos_macpong/03oos_macpong.c

```diff
@@ -109,7 +109,7 @@ void iphc_init(void) {
     macpong_vars.timerId = opentimers_create(TIMER_GENERAL_PURPOSE, TASKPRIO_IPHC);
     opentimers_scheduleIn(
         macpong_vars.timerId,   // timerId
-        1000,                   // duration
+        1000,                   // duration: longer duration is preferable for slower connections to avoid buffer overflow. 4000 was used for FSK at 25kbps wnad it was fine

```

Comment:
This timer is canceled when the a packet is generated. It should not be influenced by the modulation.